roles/container-build/tasks/main.yml

```diff
@@ -16,6 +16,7 @@
     state: installed
   become: true
   with_items:
+    - libselinux-python
```

Comment:
This needs a lookup depending on python3 or python2. see example python_pip_package_name